serde_arrow/src/test_impls/chrono.rs

```diff
@@ -73,6 +73,129 @@ fn naive_as_date64() {
         .check_nulls(&[&[false, false]]);
 }
 
+#[test]
```

Comment:
I think some tests for the string serialization would be nice (as there is some issue with the micro/nanosecond) types.

Reply:
I think it makes sense to leave out string serialization for now.